casbin/core_enforcer.py

```diff
@@ -18,6 +18,7 @@ class CoreEnforcer:
     adapter = None
     watcher = None
     rm_map = None
+    rm = None
```

Comment:
If possible, please follow the current structure of RoleManager map

Reply:
Oh I am sorry. Actually I added that line to solve a bug but I found some other way so that line is no more needed. I forgot to delete it. I'll just do it now.